site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -117,9 +117,12 @@
           <legend id="cat_label">Categories:</legend>
           <div id=id='categories-pick-list'>
             {% for category in categories %}
+                {# if diff ==4 then visible_Date is true #}
+                {% if category.diff == 4 or core.getUser().getGroup() <= 2 %}
```

Comment:
Can you explain where the 4 comes from?

Reply:
In the given screenshot where the date is **below** than current date the output is **greater** then zero
and where date is **above** than current date the output is **less** then zero

![Screenshot from 2022-12-27 14-00-14](https://user-images.githubusercontent.com/86769921/209645933-8f043832-b277-40a5-a67f-730cab608a2e.png)

![Screenshot from 2022-12-28 12-25-27](https://user-images.githubusercontent.com/86769921/209645957-2a6cbc72-10f0-4c1b-83a5-1536b9dea482.png)